gbus/abstractions.go

```diff
@@ -185,6 +187,7 @@ type Invocation interface {
 	Bus() Messaging
 	Tx() *sql.Tx
 	Ctx() context.Context
+	Logger() logrus.FieldLogger
```

Comment:
not sure whether we should expose the package specific implementation(logrus) in the interface used by handlers...what would be the migration path if we decide to move away from logrus in future releases?